compiler/src/iree/compiler/Codegen/LLVMGPU/Verifiers.cpp

```diff
@@ -81,7 +77,7 @@ verifyGPUMatmulPipeline(Operation *op,
                         ArrayRef<int64_t> workgroupSize) {
   // This verifier only applies to matmul.
   CodeGenPipeline pipeline = translationInfo.getDispatchLoweringPassPipeline();
-  if (pipeline != CodeGenPipeline::LLVMGPUMatmulSimt &&
+  if (pipeline != CodeGenPipeline::LLVMGPUTileAndFuse &&
```

Comment:
I don't expect the verifier to be strictly correct for the way that TileAndFuse splits up the tile sizes (tests are probably passing by chance). We can just drop the verifier here for now because we have an open issue for adding a proper one.